db/app/controllers/reviews.c.js

```diff
@@ -10,3 +10,15 @@ exports.getReviewById = (req, res, next) => {
       next(err);
     });
 };
+
+exports.patchReviewById = (req, res, next) => {
+  const { review_id } = req.params;
+  // const {inc_votes} = req.body
```

Comment:
if not using this commented out code please delete to keep things tidy

Reply:
Done